lib/amazon_ssa_support/instance_metadata.rb

```diff
@@ -0,0 +1,44 @@
+require 'httpclient'
+
+module AmazonSsaSupport
+  class InstanceMetadata
+    
+    def initialize(version='latest')
+      @baseUrl    = 'http://169.254.169.254/'
+      @version    = version
```

Comment:
@hsong-rh I *think* if a variable doesn't need to be an instance variable it probably shouldn't be an instance variable. @roliveri is this correct?